packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -360,6 +360,7 @@
   "Are_you_sure": "Are you sure?",
   "Are_you_sure_you_want_to_delete_your_account": "Are you sure you want to delete your account?",
   "Are_you_sure_you_want_to_disable_Facebook_integration": "Are you sure you want to disable Facebook integration?",
+  "Livechat_request_comment_when_closing_conversation": "Livechat request comment when closing conversation",
```

Comment:
- This translation is not in the right order;
- Remove `Livechat` From both string;